userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,124 @@
+
+Given an image and a one-shot detector trained on category **stop sign**, the output prediction will look like:
+
+![Stop Sign vector and prediction images](images/stop_sign_sample.jpg) 
+
+
+#### Introductory Example
+
+In this example, the goal is to **localize** where in the image instances of the category **stop sign** are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```python
+import turicreate as tc
+
+# Load the starter images
+starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
+                   'label':['stop_sign']})
+
+# Load test images
+test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'), 
+                                tc.Image('stop_sign_photo2.png')],
+                        'label':['stop_sign', 'stop_sign']})
+
+# Create a model. This step may take several hours                                      
+model = tc.one_shot_object_detector.create(starter_images, 'label')
+
+# Save predictions on the test set as a column of the test data SFrame
+test_data['predictions'] = model.predict(test_data)
+
+# Draw prediction bounding boxes on the test images
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions']) 
+
+# Evaluate the model and save the results in a dictionary
+metrics = model.evaluate(test_data)
+
+# Save the model for later use in TuriCreate
+model.save('mymodel.model')
+
+# Export for use in Core ML
+model.export_coreml('MyCustomOneShotDetector.mlmodel')
+```
+
+Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
+
+
+```python
+augmented_images = \
+	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
+
+augmented_images.explore()
+```
+
+![Explore augmented images](images/augmented_images_explore.jpg)
+
+Some examples of synthetic training data generated:
+
+![Sample augmented images](images/augmented_images_collage.jpg)
+
+*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
+
+
+To visualize the predictions made on the test set:
+
+
+```python
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
```

Comment:
We just showed this code above, I don't see why we are showing the same code again here.

Reply:
This is to support the image displayed under of the `.explore()` utility called on the `test_data` SFrame. I'll rearrange it.